src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  --color-of-background-light: #fff;
+  --color-general-moyo: #00acdc;
+  --color-general-text-moyo: #060b35;
+  --color-secondary-text-moyo: #616070;
+}
+
+.card {
+  background-color: var(--color-of-background-light);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  margin-left: 3px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-general-text-moyo);
+  margin-bottom: 4px;
+}
+
+.card__code-item {
+  color: var(--color-secondary-text-moyo);
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__feedback {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__box {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__box:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__feedback-quantity {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-general-text-moyo);
+  text-align: right;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-secondary-text-moyo);
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-general-text-moyo);
+  font-style: normal;
+}
+
+.card__button-buy {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
```

Comment:
don't add fixed width

Reply:
thank you for the correction
I will keep it in mind